sqlitecluster/SQLiteNode.cpp

```diff
@@ -756,7 +756,7 @@ bool SQLiteNode::update() {
 
```

Comment:
No particular reason. It's not really a confusing situation when this happens, it's expected. Are there clear guidelines for when to use HMMM?